### ISSUE TITLE
i18n:js:export does not create sorted files

### ISSUE THREAD

**Issue by swelther** [fnando/i18n-js#347]
The changes from PR #318 describes how to create output files with sorted keys.

Unfortunately this does not work. Even the specs fail (tried with commit 968837d6).

The reason is that `filtered_translations` is never called from the rake task.

We had problems with different fingerprints on production on multiple servers. Any idea how to fix this?

**Comment by PikachuEXE** [comment]
Do you have failed examples for this bug?
If a spec is provided I can implement it soon.

**Comment by swelther** [comment]
Sorry I had some days of vacation and just returned :)

Hmm seems I messed something up, the specs are indeed green. Dunno what I did there. Maybe one shouldn't do such things just before vacation? ;)

As far as I understand the problem is that the `filtered_translations` function (https://github.com/fnando/i18n-js/blob/master/lib/i18n/js.rb#L74) is never called during the `export` call.

It IS indeed called in this JS file (https://github.com/fnando/i18n-js/blob/master/app/assets/javascripts/i18n/filtered.js.erb#L17). But this doesn't help for me because we use only the `i18n:js:export` rake task.

If I find some time I might add a spec that covers this. At the moment we use the non-digests version of the files.
